docs/batches-and-continuations.md

```
@@ -351,7 +372,7 @@ are incremented in the same completion transaction that already decrements `Pend
 | `ChildJobId`      | string (FK)   | The waiting invocation                               |
 | `ParentJobId`     | string? (FK)  | Set for job→job edges                                |
 | `ParentBatchId`   | string? (FK)  | Set for batch→job edges                              |
-| `Trigger`         | enum          | `AllSucceeded | AllComplete`                         |
+| `Trigger`         | enum          | `Success | Failure | Complete`                       |
```

markdownlint-cli2 (0.23.0)

[warning] Table column count
Expected: 3; Actual: 5; Too many cells, extra data will be missing

(MD056, table-column-count)